app/src/main/java/de/westnordost/streetcomplete/quests/sidewalk_surface/AddSideWalkSurface.java

```diff
@@ -0,0 +1,54 @@
+package de.westnordost.streetcomplete.quests.sidewalk_surface;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.road_surface.AddRoadSurfaceForm;
+
+/**
+ * Created by Neelam Purswani on 12-01-2018.
+ */
+
+public class AddSideWalkSurface extends SimpleOverpassQuestType
```

Comment:
Swap to 'AddFootwaySurface' and remove (footway=sidewalk) so that this is a general-purpose footway surface annotator.

Also, we should explore all of the surface options to make sure we're (1) not missing anything and (2) not using any road-specific tags: https://taginfo.openstreetmap.org/keys/surface#values